Sources/Exporters/OpenTelemetryProtocolHttp/logs/OtlpHttpLogExporter.swift

```diff
@@ -14,7 +14,8 @@ public func defaultOltpHttpLoggingEndpoint() -> URL {
 public class OtlpHttpLogExporter : OtlpHttpExporterBase, LogRecordExporter {
   
   var pendingLogRecords: [ReadableLogRecord] = []
-  
+  let dispatchQueue = DispatchQueue(label: "OtlpHttpLogExporter Queue")
```

Comment:
I would prefer if you imported Locks.swift file which is in other parts of the project (e.g in the sdk, but its methods are internal) and you used it instead of GCD queues like:

`private let exporterLock = Lock()`

Reply:
It appears that Lock is internal to OpenTelemetrySdk. Would you like me to make it public, or copy it to OpenTelemetryProtocolHttp?

Reply:
Copy in the exporter folder. Thanks.

Reply:
Updated.